### ISSUE TITLE
Rename Lautus-Theme

### ISSUE THREAD

**Issue by theuserbl** [GreenteaOS/Greentea#97]
Rename the Lautus theme.
I know, it is the Lautus-Theme of ReactOS "only" with different bitmaps. But with the different bitmaps it is an other theme!
The Lautus theme is the black one theme.
The normal way would be, to let the Lautus there where it is an creating an additional theme.
But what you do is, to modify some parts of it and let the name there. But again: This is no longer the Lautus theme, so give it an other name. It would only end up in confusings.

**Comment by PeyTy** [comment]
@theuserbl Will do! Any naming proposals?

**Comment by theuserbl** [comment]
No, I haven't one. But I think something with a "7" or "8" in the name would be nice. Because - and that is something what me wonder - your new theme looks on ReactOS 0.4.6 and GreenteaOS 0.5.1 different:
![newlautus](https://user-images.githubusercontent.com/8337032/30248116-2677800e-9622-11e7-881b-0e93d610b132.png)
On ReactOS 0.4.6 it looks like Windows 8 and on GreenteaOS 0.5.1 it looks a little bit like Windows 7. (But maximize-buttons and minimize-buttons are missing on both).
And with Windows XP this theme don't work. I have create a *.theme file for it, but WinXP don't like the lautus.msstyles file.
